frontend/website/src/components/ProfileHero.re

```diff
@@ -300,7 +304,10 @@ let make = (~member: Leaderboard.member) => {
     <span className=Styles.participantDetails>
       {React.string("Participant Details")}
     </span>
-    <p className=Styles.header> {React.string(member.name)} </p>
+    <div className=Styles.nameContainer>
+      <p className=Styles.username> {React.string(member.name)} </p>
+      {Leaderboard.LeaderboardRow.DesktopLayout.renderBadges(member)}
```

Comment:
is there a reason why you put the function in DesktopLayout and not in this module?